shared/styles/style-guide.desktop.js

```diff
@@ -44,7 +44,8 @@ export const globalColors = {
 }
 
 export const globalResizing = {
-  login: {width: 700, height: 580},
+  // Add 25px to login for the OS menubar
+  login: {width: 700, height: 605},
```

Comment:
This is what `setContentSize` should account for. Let me look why that isn't working. We shouldn't have to account for the menubar ourselves.